### PR TITLE
test: increase test coverage

### PR DESCRIPTION
**Pull request by mmeloni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +62,10 @@ immuproof serve --api-key {your api key} --port 443 --host admin.cas.codenotary.
   IMMUPROOF_WEB_ADDRESS=
   IMMUPROOF_WEB_KEY_FILE=
   IMMUPROOF_WEB_CERT_FILE=
+  IMMUPROOF_WEB_TITLE_TEXT=
+  IMMUPROOF_WEB_HOSTED_BY_LOGO_URL=
+  IMMUPROOF_WEB_HOSTED_BY_TEXT=
```
Comment (tomekkolo) [inline README.md:67]: @mmeloni please add example command on how to run with docker

Comment (tomekkolo) [inline README.md:67]: @mmeloni please add example command on how to run with docker

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
 # Immuproof
 
-Simple audit tool for CAS and CodeNotaryCloud services.
+Simple audit tool for CAS service.
```
Comment (ldej) [inline README.md:3]: The 'S' in CAS stands for Service already, so no need to add the word service

File: audit/simpleAuditor_test.go [hunk 3]
```diff
@@ -16,12 +16,14 @@ package audit
...
 	"context"
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"testing"
 	"time"
 
-	"github.com/codenotary/immuproof/cnc"
+	"github.com/codenotary/immuproof/cnc/cnctest"
```
Comment (ldej) [inline audit/simpleAuditor_test.go:25]: Please group and sort imports properly. I recommend https://github.com/incu6us/goimports-reviser

File: cmd/root.go [hunk 4]
```diff
@@ -30,10 +30,13 @@ import (
...
-	Use:   "immuproof",
-	Short: "Simple audit tool for CAS and CodeNotaryCloud services",
-	Long: `Simple audit tool for CAS and CodeNotaryCloud services.
+var rootCmd = NewRootCmd()
+
+func NewRootCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "immuproof",
+		Short: "Simple audit tool for CAS service",
+		Long: `Simple audit tool for CAS service.
```
Comment (ldej) [inline cmd/root.go:39]: Remove 'service' here as well

File: rest/rest_test.go [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+	req := httptest.NewRequest("GET", "/", nil)
+	require.NoError(t, err)
+	rr := httptest.NewRecorder()
+	sh := countHandler{statusMap: srm}
+	sh.ServeHTTP(rr, req)
+	resp := rr.Result()
+	require.Equal(t, 200, resp.StatusCode)
+}
+
+func TestWevHandler_ServeHTTP(t *testing.T) {
```
Comment (ldej) [inline rest/rest_test.go:57]: TestWevHandler -> TestWebHandler

**Review by ldej — COMMENTED** [review]
Minor textual comments

File: README.md (hunk 1 above)
Comment (mmeloni) [reply README.md:67]: I will do in another branch when merged all last modifications with the new docker image

**Review by ldej — APPROVED** [review]

**Pull request merged**